services/QuillLMS/spec/controllers/api/v1/activity_sessions_controller_spec.rb

```diff
@@ -42,37 +42,45 @@
 
     context 'concept results are included' do
       let(:writing_concept) { create(:concept, name: 'Creative Writing') }
+      let(:another_concept) { create(:concept, name: 'Creative Writing') }
 
       let(:concept_result1) do
-        create(:old_concept_result,
-          activity_session_id: activity_session.id,
-          concept: writing_concept,
-          metadata: { foo: 'bar', correct: true }
-        )
+        {
```

Comment:
Is there a reason we're not using the factory to generate these concept results?

Reply:
Perhaps these should have different names, but these aren't actually `ConceptResult`-shaped hashes.  These are the shape that the front-end uses when it reports concept result data.  So what we're actually testing here is that the payloads do what we want.  Would renaming these variables be enough to make that clear, do you think?

Reply:
I think renaming would help with readability.  Otherwise, I would look at this code in a few months and ask the same question why a factory is not used here.  A different name can also help identify a possibly new object that could be abstracted later.

Reply:
Done.